### PR TITLE
delete captain account API was being added

### PR DESCRIPTION
**Pull request by ralsalloum**
- delete captain account API was being added.

### CODE REVIEW

File: backend/src/Service/Eraser/EraserService.php [hunk 1]
```diff
@@ -2,20 +2,131 @@
...
+use App\Service\CaptainPayment\CaptainPaymentService;
+use App\Service\CaptainPayment\CaptainPaymentToCompanyService;
+use App\Service\ChatRoom\OrderChatRoomService;
+use App\Service\Image\ImageService;
+use App\Service\Notification\NotificationFirebaseService;
+use App\Service\Order\OrderService;
+use App\Service\User\UserService;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
 
 class EraserService
```
Comment (Kenan-Hussein) [inline backend/src/Service/Eraser/EraserService.php:9]: please separate this class into multi classes, each class for one type of deletion

File: backend/src/Service/CaptainPayment/CaptainPaymentToCompanyService.php [hunk 2]
```diff
@@ -32,8 +32,14 @@ public function getCaptainPaymentsToCompany(CaptainPaymentFilterRequest $request
...
+    // This function was commented out because it is not being used in any place
+//    public function getSumPaymentsToCompany($captainId): array
+//    {
+//        return $this->captainPaymentManagerToCompany->getSumPaymentsToCompany($captainId);
+//    }
+
+    public function getPaymentToCompanyByCaptainId(int $captainId): array
     {
-        return $this->captainPaymentManagerToCompany->getSumPaymentsToCompany($captainId);
+        return $this->captainPaymentToCompanyManager->getPaymentToCompanyByCaptainId($captainId);
```
Comment (Kenan-Hussein) [inline backend/src/Service/CaptainPayment/CaptainPaymentToCompanyService.php:43]: can you confirm this change, please?

**Review by Kenan-Hussein — CHANGES_REQUESTED** [review]
please resolve the requested changes

File: backend/src/Service/CaptainPayment/CaptainPaymentToCompanyService.php (hunk 2 above)
Comment (ralsalloum) [reply backend/src/Service/CaptainPayment/CaptainPaymentToCompanyService.php:43]: The function which marked red is not being used in any where, so I commented it out, in coordination with @habibdevsy

**Review by Kenan-Hussein — APPROVED** [review]

**Pull request merged**